packages/data-store/src/types/uniformCredential/IAbstractUniformCredentialStore.ts

```diff
@@ -0,0 +1,29 @@
+import { CredentialCorrelationType, CredentialDocumentFormat, CredentialStateType, CredentialTypeEnum, UniformCredential } from './uniformCredential'
+
+export type GetUniformCredentialArgs = { id: string } | { hash: string }
```

Comment:
refactored to `*DigitalCredentialArgs`

Reply:
Okay that wasn't exactly what I hoped you would do. If I am already accessing the DigitalCredentialStore, I do not have to name every single method DigitalCredential (that is already implied for a Credential Store with the name Digital in it.

What you can end up is someone calling the store instance constant `digitalCredentialStore` and then you thus end up with `digitialCredentialStore.getDigitalCredential()`. Just call these methods `xxxCredential`. They are fully typed, thus it will be clear it either accepts or returns Digital Credentials. The word digital adds nothing in all the method names

Reply:
removed the digital keywords from the function names. now we have: `getCredential
getCredentials
addCredential
updateCredentialState
removeCredential`